requirements.txt

```diff
@@ -0,0 +1 @@
+coala_bears>=0.10.0
```

Comment:
That was from: https://github.com/coala/coala-sublime/pull/19/files actually. I didn't check the version when I made the PR. I'm not sure what you mean - gentle reminder for?